app/main/views/marketplace.py

```diff
@@ -187,3 +189,33 @@ def list_opportunities(framework_slug):
         # Buyers can create new briefs and want to see their updates more quickly.
         response.cache_control.max_age = min(300, current_app.config['DM_DEFAULT_CACHE_MAX_AGE'])
     return response
+
+
+@main.route('/case-studies', methods=['GET'])
```

Comment:
As discussed, react component rendering could be made more concise using a templating decorator http://flask.pocoo.org/docs/0.11/patterns/viewdecorators/#templating-decorator